### ISSUE TITLE
Use of * [ AVP ] in Subscription-Id-Extension 

### ISSUE THREAD

**Issue by dcdolson** [lbertz02/rfc4006bis#25]
As defined, *[AVP] is permitted. 
However, text says "exactly one AVP must be included" and hence more than one [AVP] would not be permitted.
I think it should simply be [AVP] as follows:
```
Subscription-Id-Extension ::= < AVP Header: TBD7 >
                             [ Subscription-Id-E164 ]
                             [ Subscription-Id-IMSI ]
                             [ Subscription-Id-SIP-URI ]
                             [ Subscription-Id-NAI ]
                             [ Subscription-Id-Private ]
                             [ AVP ]
```
Same change should be done for User-Equipment-Info-Extension and Redirect-Server-Extension.

**Comment by yuvalif** [comment]
agree. should fix in all extensions that say "exactly one"

**Comment by dcdolson** [comment]
Fixed in commit ab9c141d75fcc81136cd8afed9b5dc42e027332e
